docs/feature_split_keyboard.md

```diff
@@ -161,21 +161,22 @@ This setting is not changed when re-initializing the EEPROM using the `EE_CLR` k
 
 You can find the `EEPROM` files in the QMK firmware repo, [here](https://github.com/qmk/qmk_firmware/tree/master/quantum/split_common).
 
+
 #### Handedness by `#define`
 
-You can set the handedness at compile time.  This is done by adding the following to your `config.h` file:
+You can use this option when USB cable is always connected to either the right or left side only. Add one of the following to your `config.h` file:
 
 ```c
 #define MASTER_RIGHT
 ```
-
-or 
+QMK will assume that the USB cable will always be connected on the right side. Flash both sides with this option.
```

Comment:
I think it would be more readable if this sentence is moved before the `#define` above.